### ISSUE TITLE
bitmap scan runtime error

### ISSUE THREAD

**Issue by kainwen** [greenplum-db/gpdb#5353]
### Greenplum version or build

Current Master Branch Top Commit: f3e5c0932886773cbda3407d4ddccbd15ab304e7

### OS version and uname -a

ubuntu16.04 
This problem is not related to OS.

### Step to reproduce the behavior

This is based on the case of regress: bitmap_index

```sql
SET enable_seqscan = OFF;
SET enable_indexscan = ON;
SET enable_bitmapscan = ON;

create table bm_test (i int, t text);
insert into bm_test select i % 10, (i % 10)::text  from generate_series(1, 100) i;
create index bm_test_idx on bm_test using bitmap (i);
select count(*) from bm_test where i=1;
select count(*) from bm_test where i in(1, 2);
```

The above sql produces no errors. This is because the tuples with i = 1 and i =2 are both on the same segment.

Let's try another query:

```sql
SET enable_seqscan = OFF;
SET enable_indexscan = ON;
SET enable_bitmapscan = ON;

create table bm_test (i int, t text);
insert into bm_test select i % 10, (i % 10)::text  from generate_series(1, 100) i;
create index bm_test_idx on bm_test using bitmap (i);
select count(*) from bm_test where i=1;
select count(*) from bm_test where i in(1, 8);
```
**Attention! Tuples with i = 1 and i = 8 are on different segments!**

This leads to a runtime error.

The reason is:
 in the function `MultiExecBitmapIndexScan`, it loops the array `(1, 8)`, and for each value it gets the corresponding bitmap via `bmgetbimap`.

 If `bmgetbimap` cannot acquire a bitmap, it return an empty TIDBITMAP. This will cause an error in the next time loop in `MultiExecBitmapIndexScan`.

**Comment by hlinnaka** [comment]
Any progress on this? I ran into this yesterday, while working on an unrelated patch.

**Comment by kainwen** [comment]
@hlinnaka 
That PR should have fixed the issue but not providing an empty streambitmap create & test API. And that is the reason we close that. 

MPP team has been busy with OnlineExpand, so that story has been transferred to other team.

And @paul-guo- said to me recently, he takes this issue.

**Comment by kainwen** [comment]
@paul-guo- 
@hlinnaka 
How about we let that PR in and open a new issue that "add A complete API to build empty streambitmap"?

**Comment by yydzero** [comment]
@gaos1 FYI

**Comment by kainwen** [comment]
Fixed by the commit cbf622de4dc64deddaaa9e33db6f674a58c2d508
